src/dewey/indexing.clj

```diff
@@ -125,13 +130,12 @@
 
    Throws:
      This function can throw an exception if it can't connect to elasticsearch."
-  [es entity-type entity-id]
-  (when (entity-indexed? es entity-type entity-id)
-    (es-doc/delete es (cfg/es-index) (mapping-type-of entity-type) (str entity-id))))
-
+  [es entity-id]
+  (when (entity-indexed? es (str entity-id))
+    (s/request es {:url [(cfg/es-index) :_doc (str entity-id)]})))
```

Comment:
This looks incorrect to me -- shouldn't this need to specify the `DELETE` method somehow? It looks from the spandex docs as though the default is `GET` as it usually would be.

Reply:
You're right, this does look wrong. I'm not sure what happened to this code, but I will update this.